app/javascript/src/apis/contact.js

```diff
@@ -28,11 +28,18 @@ const fetch = async () => {
     },
   };
 };
-const create = async () => {};
+const create = async values => {
+  values;
+};
+const update = async values => {
+  values;
+};
```

Comment:
There's no point in providing `async` if the value is not a Promise that is resolved.

Reply:
I corrected this in the previous [PR](https://github.com/navaneethsdk/wheel/pull/12#discussion_r978714762). Kindly look into it.